### DISCUSSION TITLE
I have not received my education benefits with my approval status is approved

### DISCUSSION THREAD

**Discussion by ChenLX233** [community/community#148058] [GitHub Education]
### Hi everyone,

I have not received my education benefits after being approved
And I have not received any email
![Uploading 屏幕截图 2024-12-28 124449.png…]()

**Comment by davevad93** [comment]
Hi @ChenLX233 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ChenLX233 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by ChenLX233** [comment]
When the benefits become available, the status label will turn purple and change to:
•	Congrats! Your request for @XXX was approved and your benefits are now available.
The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

**Comment by ChenLX233** [comment]
Thank you very much for your help. I am sorry for my negligence.


________________________________
寄件者: Davide Presti ***@***.***>
寄件日期: 星期六, 12月 28, 2024 7:42 下午
收件者: community/community ***@***.***>
副本: Lingxuan Chen ***@***.***>; Mention ***@***.***>
主旨: Re: [community/community] I have not received my education benefits with my approval status is approved (Discussion #148058)


Hi @ChenLX233<https://github.com/ChenLX233> ,

When the benefits become available, the status label will turn purple and change to:

  *   Congrats! Your request for @ChenLX233 was approved and your benefits are now available.

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions:
#132790<https://github.com/orgs/community/discussions/132790> , #131665<https://github.com/orgs/community/discussions/131665> , #132470<https://github.com/orgs/community/discussions/132470> , #133531<https://github.com/orgs/community/discussions/133531>, #134279<https://github.com/orgs/community/discussions/134279>, #135849<https://github.com/orgs/community/discussions/135849>, #136887<https://github.com/orgs/community/discussions/136887>, #138721<https://github.com/orgs/community/discussions/138721>, #139268<https://github.com/orgs/community/discussions/139268>, #140287<https://github.com/orgs/community/discussions/140287>, #145640<https://github.com/orgs/community/discussions/145640>, #145995<https://github.com/orgs/community/discussions/145995>

And also the Student Developer Pack Application FAQs<https://github.com/orgs/community/discussions/111352> :

  *   How long after I submit my application will it be approved?<https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved>
  *   How long after I’ve been approved will I receive my academic benefits?<https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits>

**Reply by davevad93** [reply]
Hey @ChenLX233 don't worry and you're welcome! 👍
